specification/loadtestservice/resource-manager/Microsoft.LoadTestService/playwright/readme.typescript.md

```diff
@@ -0,0 +1,14 @@
+## TypeScript
```

Comment:
I think we can remove the readme.language.md file if there's typespec in this service.

Reply:
done